share/ci/bash.profile

```diff
@@ -22,6 +22,9 @@ export CMAKE_PREFIX_PATH=$ADIOS2_ROOT:$CMAKE_PREFIX_PATH
 export PATH=$ADIOS2_ROOT/bin:$PATH
 export LD_LIBRARY_PATH=$ADIOS2_ROOT/lib:$LD_LIBRARY_PATH
 
+# set environment variable for path to tpls for PyPIConGPU runner
```

Comment:
What has this change to do with LWFA example or ionization methods?

Reply:
the CI does not set a `PIC_SYSTEM_TEMPLATE_PATH` by default causing the PICMI run time tests to fail due to the switch to using the bash environment for `tpl` file resolution in the runner. It was therefore necessary to add setting it by hand to the CI-environment init script.